.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+* @taka-sho
+
+/tagupy @takesei @yo-aka-gene @gk-hazard
```

Comment:
I set the CODEOWNER.
- @taka-sho will be responsible for the whole code
- @yo-aka-gene @gk-hazard  will be responsible for DoE content

When someone tries to edit the code and send the PR, people who are assigned as CODEOWNER need reviewing.